src/Config.js

```diff
@@ -501,6 +505,18 @@ export class Config {
     }
   }
 
+  static validateLogLevels(logLevels) {
+    for (const key of Object.keys(LogLevels)) {
+      if (logLevels[key]) {
+        if (ctrlLogLevels.indexOf(logLevels[key]) === -1) {
+          throw key + ' must be one of theses ' + ctrlLogLevels;
```

Comment:
```suggestion
          throw `${key} must be one of ${JSON.stringify(ctrlLogLevels)}`;
```

If this is an array, maybe we need to stringify for proper log output?